src/test/java/com/epam/steps/admin/CreateParentSteps.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.steps.admin;
+
+import com.epam.steps.BaseSteps;
+import io.cucumber.java.en.Then;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CreateParentSteps extends BaseSteps {
+
+    @Then("Check parent created by admin is not added in the DB")
+    public void checkParentIsNotAddedInTheDB() {
+        assertThat(dbHelper.isUserAddedInTheDB())
+                .withFailMessage("Parent wasn't meant to be added in the DB, but was added.")
+                .isTrue();
+    }
+
+    @Then("Check parent created by admin is added in the DB")
+    public void checkParentIsAddedInTheDB() {
+        assertThat(dbHelper.isUserAddedInTheDB())
+                .withFailMessage("Parent wasn't meant to be added in the DB, but was added.")
```

Comment:
Please, change the fail message, because in this case parent was meant to be added in the DB, but in case of fail it will not be added. You wrote the opposite.